presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -114,6 +114,7 @@
     private boolean assumeCanonicalPartitionKeys;
 
     private boolean useOrcColumnNames;
+    private boolean orcBloomFiltersEnabled = true;
```

Comment:
We could do something like add another option, `hive.orc.bloom-filters.assume-utf8`